README.md

```diff
@@ -46,6 +46,29 @@ GET example.com/api/v1/blogs
 # note: you can use any fields from the resource as filter
 # and you can use as many filters as you want.
 GET example.com/api/v1/blogs/?category=tech
+
+# Filtering with operator
+# You can use operator in filtering requests
+# by indicating the field that you want to filter,
+# the operator and the value
+
+# Get collection where views > 10
+GET example.com/api/v1/blogs/?views[gt]=10
+
+# Get collection where id not equal to 2
+GET example.com/api/v1/blogs/?id[neq]=2
+
+# Get collection where id is between 2 and 200
+GET example.com/api/v1/blogs/?id[gte]=2&id[lte]=200
+
+# Supporting operators:
+## 'gte': greater than and equal (>=)
+## 'gt': greater than (>)
+## 'lte': lesser than and equal (<=)
+## 'lt': lesser than (<)
+## 'eq': equal to (=) - used as default if not specified
+## 'neq': not equal to (<>)
+## 'like': like
```

Comment:
How about using `contains` instead of `like` since `like` is a SQL term?

Reply:
Changed to `contains`